clusterctl/README.md

```diff
@@ -0,0 +1,64 @@
+# clusterctl
+
+`clusterctl` is the SIG-cluster-lifecycle sponsored tool that implements the Cluster API.
+
+Read the [experience doc here](https://docs.google.com/document/d/1-sYb3EdkRga49nULH1kSwuQFf1o6GvAw_POrsNo5d8c/edit#).
+
+## Getting Started
+
+### Prerequisites
+
+Follow the steps listed at [CONTRIBUTING.md](https://github.com/kubernetes-sigs/cluster-api/blob/master/cluster-api/clusterctl/CONTRIBUTING.md) to:
+
+1. Build the `clusterctl` tool
+
+```
+go build
+```
+ 
+2. Create a `machines.yaml` file configured for your cluster. See the provided template for an example.
+
+### Limitations
+
+
+### Creating a cluster
+
+**NOT YET SUPPORTED!**
+
```

Comment:
I would cut out everything other than **NOT YET SUPPORTED!** in the creating a cluster section till it actually is supported. Similar reasoning to flags, it is best to keep the implementation and documentation in sync.

Reply:
Fair.